src/encoded/commands/create_mapping_on_deploy.py

```diff
@@ -123,6 +123,85 @@
     'Page',
 ]
 
+WEBPROD = 'fourfront-webprod'
+WEBPROD_TWO = 'fourfront-webprod2'
+MASTERTEST = 'fourfront-mastertest'
+HOTSEAT = 'fourfront-hotseat'
+WEBDEV = 'fourfront-webdev'
+
+
+BEANSTALK_PROD = [
+    WEBPROD,
+    WEBPROD_TWO,
+]
+
+BEANSTALK_TEST = [
+    MASTERTEST,
+    HOTSEAT,
+    WEBDEV,
+]
+
+FF_DEPLOY_CONFIG_SIZE = 2
+
+class DeploymentConfig(object):
+    """
+        Basic class wrapping an arbitrary number of configuration options
+        The caller must know the structure and how to interpret it
```

Comment:
Must it? Can't we just use a dictionary and all-keyword calls? You can keyword-call even positional args.

Reply:
Replaced